docs/ru/7-runtime.md

```diff
@@ -198,11 +203,13 @@ applyCtx(bemjson, newctx)
 Шаблон:
 
 ```js
-block('header').def()(function() {
-    return applyCtx(this.extend(this.ctx, {
-        block: 'layout',
-        mix: [{ block: 'header' }].concat(this.ctx.mix || [])
-    }));
+block('header')({
+    default: function() {
```

Comment:
It won't work in few IEs, right? Cuz of it's a protected reserved word: https://msdn.microsoft.com/en-us/library/ttyab5c8.aspx

Ie8 for sure, not sure about ie9-11.

Reply:
@zxqfox anyway it work if you use quotes ;)